kedro/framework/cli/utils.py

```diff
@@ -31,6 +31,7 @@
     "line_magic": "kedro.line_magic",
     "hooks": "kedro.hooks",
     "cli_hooks": "kedro.cli_hooks",
+    "starter": "kedro.starter"
```

Comment:
```suggestion
    "starters": "kedro.starters"
```
Just seems more natural to me (like `hooks`).

Not sure whether this should go inside some hierarchy like `kedro.constants.` as you suggested before (but then what else might go in there?) or whether it belongs at the top level 🤔

Reply:
I hesitated for the "s", let's go for the suggestions. I know we originally discussed about calling it "constants", but I make very strong assumptions on the format when looping on "starters" entrypoint, so This is clearly not generic enough to accept other constants (+ I do not see any other use case as you mention)

Reply:
Makes sense. I'm happy to have it as a top-level entry point `kedro.starters`.

Reply:
Oh, and I've just remembered why I choose to call it "starter" instead of "starters": it is the name tof the CLI group command (``kedro starter list``) so I wanted to be consistent. What do you want to do? Keep the "s" or not?

Reply:
Ah makes sense. Personally I'd go for `starters` since I wouldn't be too surprised if `kedro starter` at some point in the future somehow gets subsumed into `kedro new`, and the inconsistency here makes grammatical sense to me. But let's see what someone else thinks too - @lorenabalan?